FreeOTP.xcodeproj/project.pbxproj

```diff
@@ -56,23 +52,8 @@
 		};
 /* End PBXContainerItemProxy section */
 
-/* Begin PBXCopyFilesBuildPhase section */
-		F1CA93D21B56AA8400828B8B /* Embed Frameworks */ = {
-			isa = PBXCopyFilesBuildPhase;
-			buildActionMask = 2147483647;
-			dstPath = "";
-			dstSubfolderSpec = 10;
-			files = (
-				F1CA93D11B56AA8400828B8B /* Base32.framework in Embed Frameworks */,
-				F1CA93CF1B56AA8400828B8B /* moa.framework in Embed Frameworks */,
-			);
-			name = "Embed Frameworks";
-			runOnlyForDeploymentPostprocessing = 0;
-		};
-/* End PBXCopyFilesBuildPhase section */
-
 /* Begin PBXFileReference section */
-		F10D88C01B56A3C400482D15 /* FreeOTP.app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = FreeOTP.app; sourceTree = BUILT_PRODUCTS_DIR; };
+		F10D88C01B56A3C400482D15 /* FreeOTP (dev).app */ = {isa = PBXFileReference; explicitFileType = wrapper.application; includeInIndex = 0; path = "FreeOTP (dev).app"; sourceTree = BUILT_PRODUCTS_DIR; };
```

Comment:
Why did this change?

Reply:
Because Xcode wouldn't let me install it to my phone with the same app/bundle name, and it's impossible to test the camera in the simulator.  All of these were fixed in a later changeset, but I'll flatten it all into a new pull request.